relay-server/src/metrics_extraction/snapshots/relay_server__metrics_extraction__event__tests__extract_span_metrics.snap

```diff
@@ -48,7 +48,7 @@ expression: metrics
             1.0,
         ),
         tags: {
-            "span.op": "mYOp",
+            "span.op": "myop",
```

Comment:
This tag was previously transferred manually, so it did not go through regular tag normalization (i.e. lowercasing).